pages/balance-budget.vue

```diff
@@ -267,31 +183,79 @@ import FiscalYearSelect from "@/components/FiscalYearSelect";
 import Header from "@/components/Header";
 import Footer from "@/components/Footer";
 import DepartmentsWalkthrough from "@/components/DepartmentsWalkthrough";
+import { D3PieChart } from 'vue-d3-charts';
 
+var total_expenses = 1234.0
+var starting_expense = total_expenses / 7.0
+var current_sum_expenses = 1234.0
```

Comment:
for variables that can change, we prefer `let`. Also shouldn't this just be the starting total expenses?
```suggestion
let current_sum_expenses = total_expenses
```